client/src/components/Paypal Integration.txt

```diff
@@ -0,0 +1,17 @@
+Steps to paypal integration:
```

Comment:
could you move this to a `PROJECT_ROOT/docs` folder

Reply:
Got it